pb_project.yaml

```diff
@@ -36,4 +36,4 @@ id_types:
       - type: exclude
         value: "test@company.com"
 python_requirements:
-  - profiles-pycorelib==0.2.1
+  - profiles-pycorelib==0.2.2
```

Comment:
This is ok. I am wondering if this is also the right place to put version constraints for rudderstack-profiles. CC: @nidhilashkari17 @debanjan97 @arpl

Reply:
this allows for flexibility but the question is whether we want to expose them to the customer or hide them in our delivery process.